src/views/RecIM/components/RecimBracket/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { Bracket, IRoundProps } from 'react-brackets';
+
+const rounds: IRoundProps[] = [
+  {
+    title: 'Round one',
+    seeds: [
+      {
+        id: 1,
+        date: new Date().toDateString(),
+        teams: [{ name: 'Team A' }, { name: 'Team B' }],
+      },
+      {
+        id: 2,
+        date: new Date().toDateString(),
+        teams: [{ name: 'Team C' }, { name: 'Team D' }],
+      },
+    ],
+  },
+  {
+    title: 'Round two',
+    seeds: [
+      {
+        id: 3,
+        date: new Date().toDateString(),
+        teams: [{ name: 'Team A' }, { name: 'Team C' }],
+      },
+    ],
+  },
+];
+
+const RecimBracket = () => {
+  return <Bracket rounds={rounds} />;
+};
+
+export default RecimBracket;
```

Comment:
```suggestion
const RecIMBracket = () => {
  return <Bracket rounds={rounds} />;
};

export default RecIMBracket;
```

Sorry this is going to annoy me